config-scripts/vortex-symlinks.sh

```diff
@@ -65,21 +63,18 @@ find_current_game_paths () {
     CURRENT_INSTALL=""
     CURRENT_PREFIX=""
 
-    if [ -d "$STEAM_PROTON1_PATH/steamapps/compatdata/$CURRENT_APPID/pfx" ]; then
-        CURRENT_INSTALL="$STEAM_PROTON1_PATH/steamapps/common/$CURRENT_GAMEDIR"
-        CURRENT_PREFIX="$STEAM_PROTON1_PATH/steamapps/compatdata/$CURRENT_APPID/pfx"
-        CURRENT_GAME_USER="steamuser"
-
-    elif [ -d "$STEAM_PROTON2_PATH/steamapps/compatdata/$CURRENT_APPID/pfx" ]; then
-        CURRENT_INSTALL="$STEAM_PROTON2_PATH/steamapps/common/$CURRENT_GAMEDIR"
-        CURRENT_PREFIX="$STEAM_PROTON2_PATH/steamapps/compatdata/$CURRENT_APPID/pfx"
-        CURRENT_GAME_USER="steamuser"
-
-    elif [ -d "$WINESTEAM_PATH/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/$CURRENT_GAMEDIR" ]; then
+    if [ -d "$WINESTEAM_PATH/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/$CURRENT_GAMEDIR" ]; then
         CURRENT_INSTALL="$WINESTEAM_PATH/prefix64/drive_c/Program Files (x86)/Steam/steamapps/common/$CURRENT_GAMEDIR"
         CURRENT_PREFIX="$WINESTEAM_PATH/prefix64"
         CURRENT_GAME_USER=$USER
 
+    else
+    	LIBRARY_PATH=$(${0%/*}/../utils/find-library-for-appid.sh $CURRENT_APPID)
```

Comment:
Here you can use the variable `VORTEX_PREFIX`.
In sh, double quoting paths is important to avoid problems with paths containing spaces.

```suggestion
    	LIBRARY_PATH=$("$VORTEX_PREFIX/utils/find-library-for-appid.sh" $CURRENT_APPID)
```